opensearch-operator/pkg/builders/cluster.go

```diff
@@ -194,10 +194,15 @@ func NewSTSForNodePool(
 
 	var mainCommand []string
 	com := "./bin/opensearch-plugin install --batch"
-	if cr.Spec.General.Monitoring {
-		clusterVersion := cr.Spec.General.Version
-		url := "https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/" + clusterVersion + ".0/prometheus-exporter" + clusterVersion + ".0.zip"
-		cr.Spec.General.PluginsList = append(cr.Spec.General.PluginsList, url)
+	if cr.Spec.General.Monitoring.Enable {
+		if cr.Spec.General.Monitoring.OfflinePlugin == "" {
```

Comment:
Condition is the wrong way. then block should be entered if `OfflinePlugin != ""`